src/Server.php

```diff
@@ -20,12 +19,12 @@ interface Server
      * implementations to prevent naming collisions.
      *
      * @param  string $function
-     * @param  string $namespace
+     * @param  null|string $namespace
      * @param  null|array Optional array of arguments to pass to callback at
      *                    dispatch.
      * @return void
      */
-    public function addFunction($function, $namespace = '');
+    public function addFunction(string $function, ?string $namespace = null): void;
```

Comment:
signature change - default value, here and the next couple more methods below

if it is something needed it should be documented in migration guide

Reply:
Note has been added to the migration guide.